cli/deployment/doc.go

```diff
@@ -0,0 +1,7 @@
+/*
+Package Deployment implements GCP Deployment related functionality:
+sort deployment configs in dependency order,
+execute API calls to create, update, delete, describe deployment using gcloud tool
+ */
```

Comment:
"The deployment packages implements functionality for deploying projects to GCP. This includes sorting deployment configurations in dependency order, and executing GCP API calls through the gcloud tool."